package.json

```diff
@@ -36,12 +36,15 @@
     "@babel/polyfill": "^7.4.4",
     "@babel/preset-env": "^7.2.3",
     "@babel/preset-react": "^7.0.0",
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.4.0",
+    "@testing-library/react-hooks": "^3.2.1",
     "babel-core": "^7.0.0-bridge.0",
     "babel-eslint": "^10.0.1",
     "babel-jest": "^23.6.0",
     "babel-loader": "^8.0.5",
     "css-loader": "^2.1.0",
-    "cypress": "^3.1.4",
+    "cypress": "3.8.0",
```

Comment:
Also, unless we have a specific reason for doing so: we want to allow for any MINOR and PATCH upgrades to our dependencies.

Something like: `^3.8`. This would set a lower bound to what we think we need now and an upper bound to future "backwards compatible" versions so our dependencies tree is a bit more loose.